.github/workflows/self-zizmor.yml

```diff
@@ -0,0 +1,26 @@
+name: zizmor GitHub Actions static analysis
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    paths:
+      - .github/workflows/reusable-zizmor.yml
+      - .github/workflows/self-zizmor.yml
```

Comment:
Just checking, is this just a test or what's the point of limiting this to these paths?

Reply:
I forgot. Probably a test. I'm going to drop the filters and run this all the time!